### ISSUE TITLE
JSON key error

### ISSUE THREAD

**Issue by HuiiBuh** [koxudaxi/datamodel-code-generator#111]
**Describe the bug**
I want to create pydantic models for the Spotify api and tried to parse an example response (see [here](https://gist.github.com/HuiiBuh/431782a7d5962d3f15b3ee068add226f)). 
+ datamode-codegen 0.4.4
+ python 3.8 

**To Reproduce**

Download the gist, install datamodel-codegen and run the following command.

The command I am using to generate the the models is
```bash
datamodel-codegen --input-file-type jsonschema --input api_response.json --out output.py
``` 

**Screenshots**

![image](https://user-images.githubusercontent.com/46561026/78471138-456a8b00-772f-11ea-968a-5699388cfdf8.png)

**Comment by koxudaxi** [comment]
@HuiiBuh 

Thank you for creating this issue.

I found about Spotify's json data.
https://labs.spotify.com/2015/03/09/understanding-spotify-web-api/

This Json data is `Json format` which isn't `Json Schema`
This code generator supports `Json Schema` or `OpenAPI`. 

About `Json Schema`
https://json-schema.org/

**Comment by HuiiBuh** [comment]
I totally missed that.  
After I converted the json data to a json scheme it is still not working. 
This time I get this exception.

![image](https://user-images.githubusercontent.com/46561026/78504084-140fb080-776b-11ea-8021-898570276f68.png)

**Comment by koxudaxi** [comment]
@HuiiBuh 

I have fixed the bug and released new version as `0.4.5`

**Comment by HuiiBuh** [comment]
Tank you very much :D
I just confirmed that it is working now.
